schemas/user.schema.js

```diff
@@ -7,8 +7,8 @@ const userSchema = new Schema({
     email: {type: String, required:[true, "Email is a required field"]},
     password: {type: String, required:[true, "Password is a required field"]},
     profile: {type: String},
-    isAdmin: {type: Boolean, default: False},
-    isApproved: {type: Boolean, default: False},
+    isAdmin: {type: Boolean, default: false},
+    isApproved: {type: Boolean, default: false},
```

Comment:
Nice catch here, however going forward it would be best to only alter the files you create, or if there is a change needed for a file you haven't created or worked on, notify the person creating, or working on it. This is because if it is an active file being worked on, it will cause a merge conflict. since git is not sure which code to use.

Reply:
Ok, noted.

Reply:
Also, 'False' with a capital 'F' does not work, because the values are case sensitive. Hence the change.

Reply:
yeah will remove some columns from the schema and that will be one of them, we will be using enums instead as per DB Schema file